api/ca.pb.go

```diff
@@ -728,7 +728,7 @@ func (p *raftProxyCAServer) GetRootCACertificate(ctx context.Context, r *GetRoot
 	defer func() {
 		if err != nil {
 			if connErr, ok := err.(transport.ConnectionError); ok {
-				if connErr == transport.ErrConnClosing {
+				if connErr == transport.ErrConnClosing || connErr == grpc.ErrClientConnClosing {
```

Comment:
> A value x of non-interface type X and a value t of interface type T are comparable when values of type X are comparable and X implements T. They are equal if t's dynamic type is identical to X and t's dynamic value is equal to x.

`grpc.ErrClientConnClosing` is an `error` interface (`errors.New` returns `error`), and `transport.ErrConnClosing` is a concrete type that implements `error`.